services/common/src/components/reports/ReportFilesTable.tsx

```diff
@@ -26,17 +26,22 @@ export const ReportFilesTable: FC = () => {
   //Dummy data
   const files = [
     {
+      document_manager_guid: "1",
       file_name: "Report 1",
       "file_type ": "PDF",
       created_by: "admin",
     },
   ];
```

Comment:
Is this dummy data a permanent thing ?

Reply:
it shouldn't be. It just bothered me that the rowKey wasn't set to an attribute, so that's the only change I made here. I think we're going separate ticket.